pkgs/applications/backup/mastodon-archive/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, fetchFromGitHub
+, buildPythonApplication
+, html2text
+, mastodonpy
+, progress
+}:
+
+buildPythonApplication rec {
+  pname = "mastodon-archive";
```

Comment:
is there any tests for this?

Reply:
None that I can see here: https://alexschroeder.ch/cgit/mastodon-archive/tree/ . Would likely require to run a mastodon server or some sort of mocking.